quantities/parser_body.hpp

```diff
@@ -216,12 +216,16 @@ inline Unit ParseQuotientUnit(std::string const& s) {
     // Not a quotient.
     return ParseProductUnit(s);
   } else {
-    // A quotient.  Parse each half.
+    // A quotient.  Parse each half.  Note that there may not be a left half for
+    // input like 1.23 / s.
```

Comment:
Update the grammar in the `hpp` and in https://github.com/mockingbirdnest/Principia/wiki/Principia-configuration-files.